etl/xml_importer/entities/location.py

```diff
@@ -44,13 +50,36 @@ def _parse_label(self):
         else:
             self.label = ""
 
+    def _parse_alt_labels(self):
+        self.alt_labels = []
+        alt_label_root = self.root.findall(paths["Location_Label_Path"], namespace)
+        if len(alt_label_root) > 1:
+            for element in alt_label_root[1:]:
+                self.alt_labels.append(sanitize(element.text))
+
+    def _inventoryNumber(self):
+        inventoryNumber = self.root.find(paths["Location_inventoryNumber"], namespace)
```

Comment:
Rename this to `inventoryNumber_root` or `inventory_number_root` and the path should be called `Location_InventoryNumber_Path`
```suggestion
        inventoryNumber_root = self.root.find(paths["Location_InventoryNumber_Path"], namespace)
```